code/backend/batch/utilities/helpers/AzureSearchIVDatasourceHelper.py

```diff
@@ -0,0 +1,47 @@
+from azure.search.documents.indexes.models import (
```

Comment:
How about a new folder in utilities called `integrated_vectorization` with these new classes and try and get rid of Helper and the long name?

Reply:
Agreed, Updated the names